### PR TITLE
refactor: extract random value generation

### PR DESCRIPTION
**Pull request by eliias**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -38,42 +37,53 @@ export class Konfettikanone extends Component {
...
+      speed
+    };
+  }
+
+  static pickRandomElement(a) {
+    return a[Konfettikanone.random(a.length)]
   }
+
+  static random(n = 1) {
+    return Math.floor(Math.random() * n);
```
Comment (helloneele) [inline src/index.js]: Can we use `Math.round()` with `a.length - 1` instead?
If we use `Math.floor()` and `Math.random()` evaluates to `1` we will get an out of range exception.

Comment (helloneele) [reply src/index.js]: Could we also add tests for this?

Comment (eliias) [reply src/index.js]: > Can we use `Math.round()` with `a.length - 1` instead?
> If we use `Math.floor()` and `Math.random()` evaluates to `1` we will get an out of range exception.

Nice catch!

**Comment by eliias** [comment]
@helloneele please review

**Pull request merged**